CHANGELOG.md

```diff
@@ -13,6 +13,7 @@
 * [ENHANCEMENT] Scraping: support Created-Timestamp feature on native histograms. #14694
 * [BUGFIX] PromQL: Fix stddev+stdvar aggregations to always ignore native histograms. #14941
 * [BUGFIX] PromQL: Fix stddev+stdvar aggregations to treat Infinity consistently. #14941
+* [BUGFIX] OTLP receiver: Preserve colons when generating metric names in suffix adding mode. #15251
```

Comment:
```suggestion
* [BUGFIX] OTLP receiver: Preserve colons when generating metric names. #15251
```

The addition of suffixes is hardcoded, so it's always "suffix adding mode".

Reply:
It isn't hard coded when using Prometheus in library mode though, for example in Mimir. Mimir can use both modes.

Reply:
Maybe I should modify the entry to clarify that it's hard coded unless using Prometheus as a library?